Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.iOS/project.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dependencies": {
+    "Microsoft.Azure.Mobile": "0.9.0",
```

Comment:
That comment was to harmonize with all platforms since explicit core dependency is not needed.